roles/installer/tasks/tower_custom_settings.yml

```
@@ -0,0 +1,11 @@
+---
+- name: Retrieve ConfigMap
+  community.kubernetes.k8s_info:
```

```suggestion
  k8s_info:
```

The FQCN shouldn't be needed here. I recently went through and removed all of these occurrences. We're going to be moving to `kubernetes.core` in the near future.